packages/edit-post/src/components/visual-editor/index.js

```diff
@@ -43,9 +49,11 @@ function VisualEditor() {
 				<CopyHandler>
 					<WritingFlow>
 						<ObserveTyping>
-							<div className="edit-post-visual-editor__post-title-wrapper">
-								<PostTitle />
-							</div>
+							{ ! templateZoomOut && (
```

Comment:
For now, this is fine if we just want to see how the feature feels, but in the long run we should absorb this exception. Off the top of my head, we might altogether remove `PostTitle` from here and ensure that, within `BlockList`, there is a `PostTitle` at the head. I don't know if this is an acceptable "breaking" change in terms of DOM and classes — though, IMO, these have always been private interfaces.

Reply:
I tried it, there's another range of issues to deal with when we do that :P